.changeset/tasty-buses-jog.md

```diff
@@ -0,0 +1,114 @@
+---
+'@keystone-next/keystone': major
+---
+
+The names of two-sided many-many relationships now only include the list key and field key for one side of the relationship and one-sided many relationships(which are many-many) no longer have `_many` at the end of them for consistency with two-sided relationships. This allows having many-many relationships with long list and field keys without hitting Postgres's 63 character limit.
+
+There are two ways to update:
+
+1. Enable the `experimental.useLegacyManyRelationNames` option at the root of your config. This will make Keystone revert to the old relation names. This option will be removed in a future release.
```

Comment:
```suggestion
1. Set `experimental.useLegacyManyRelationNames: true` at the root of your config. This will make Keystone revert to the old relation names.  This feature will be removed in a future release.
```

Reply:
While this feature will be removed in the future,  we are looking at the capability to use custom relation names rather than these automatically generated names.  This should resolve the problem in a robust manner.